napari_folder_browser/_dock_widget.py

```diff
@@ -1,113 +1,202 @@
+from pathlib import Path
+from PyQt5.QtCore import QModelIndex
```

Comment:
Could we depend on qtpy instead of PyQt5? That's important because of licensing issues. PyQt5 does not allow is to lice se our depending code BSD...

Reply:
@haesleinhuepf Certainly, that was just an oversight! Just changed it (also fixed the multi-select, have a try)